packages/perseus/src/widgets/interactive-graphs/types.ts

```diff
@@ -52,6 +52,7 @@ export interface LinearSystemGraphState extends InteractiveGraphStateCommon {
 export interface PointGraphState extends InteractiveGraphStateCommon {
     type: "point";
     coords: Coord[];
+    numPoints?: number | "unlimited";
```

Comment:
Since the number of points is given by `coords.length` for graphs with fixed numbers of points, could this property just be a boolean? E.g. `unlimitedPoints: boolean`?

In any case, I think this property shouldn't be optional, since it's always set in `initializeGraphState`.

Reply:
I think I'd prefer to keep numPoints. It matches the state we receive and I don't think there are going to be any more places where it would get tricky.